package.json

```diff
@@ -151,6 +151,7 @@
     "eslint-plugin-react": "^7.11.1",
     "jest": "^29.5.0",
     "jest-canvas-mock": "^2.2.0",
+    "jest-each": "^29.7.0",
```

Comment:
What is that for?

Reply:
For parametrized tests, I noticed the processUpload tests are almost going to match exactly the 10x ones, so I am planning to use this. [Here are some examples of how they work](https://medium.com/@suman.maity112/parameterized-tests-with-jest-2932f42b1509#:~:text=describe(%22isPalindrome%22%2C%20()%20%3D%3E%20%7B%0A%20%20each(%5B%0A%20%20%20%20%5B%22madam%22%2C%20true%5D%2C%0A%20%20%20%20%5B%22car%22%2C%20false%5D%0A%20%20%5D).it(%22when%20the%20input%20is%20%27%25s%27%22%2C%20(text%2C%20expected)%20%3D%3E%20%7B%0A%20%20%20%20expect(isPalindrome(text)).toBe(expected)%3B%0A%20%20%7D)%3B%0A%7D)%3B)